contracts/strategies/ltip-hedgey/README.md

```diff
@@ -0,0 +1,108 @@
+# LTIPHedgeyStrategy.sol
+
+The `LTIPHedgeyStrategy` contract represents a smart contract for Long-Term Incentive Programs (LTIP). It extends the capabilities of the `LTIPSimpleStrategy` contract and replaced the TokenTimeLock vesting with Hedgey vesting. The contract also incorporates the `ReentrancyGuard` library to prevent reentrant attacks.
+
+## Table of Contents
+
+- [RFPSimpleStrategy.sol](#rfpsimplestrategysol)
+  - [Table of Contents](#table-of-contents)
+  - [Sequence Diagram](#sequence-diagram)
+  - [Smart Contract Overview](#smart-contract-overview)
+    - [Structs](#structs)
+    - [Errors](#errors)
+    - [Events](#events)
+    - [Storage Variables](#storage-variables)
+    - [Constructor](#constructor)
+    - [Initialize Function](#initialize-function)
+    - [Views](#views)
+    - [External Functions](#external-functions)
+    - [Internal Functions](#internal-functions)
+  - [User Flows](#user-flows)
+    - [Registering a Recipient](#registering-a-recipient)
+    - [Setting Milestones](#setting-milestones)
+    - [Submitting a Milestone Proof](#submitting-a-milestone-proof)
+    - [Rejecting a Pending Milestone](#rejecting-a-pending-milestone)
+    - [Updating Max Bid](#updating-max-bid)
+    - [Distributing Milestone](#distributing-milestone)
+    - [Withdrawing Funds from Pool](#withdrawing-funds-from-pool)
+
+## Sequence Diagram
+
+## Sequence Diagram
+
+```mermaid
+sequenceDiagram
```

Comment:
Same improvements on the diagram as LTIPSimple i think would be cleaner.
- review step
- hedgey plan creation